src/middleware/isValidInvite.js

```diff
@@ -1,22 +1,42 @@
 const jwt = require('jsonwebtoken')
 const config = require('../config')
-const { AppError } = require('../lib')
 
 
 const verify = (req, res, next) => {
 	const authHeader = req.headers.authorization
 
-	if (!authHeader) return next(new AppError('No token provided', 401))
+	if (!authHeader)  {
```

Comment:
@jocrah yeah. it's not being handled.

Reply:
@jocrah should we handle it in a way that the test NODE_ENV should default to development?

Reply:
no, it's the issue is already handled in the most recent pr I merged actually so you can check it out and also test if it solves the problem you were facing

Reply:
@jocrah It doesn't solve it

Reply:
@itzgeebee it works at my end though, i pulled your branch and reverted just the isValidInvite file to the version on the `main` branch and it doesn't time out.

its just about how the errors are being checked that would have to change a bit since the dev error gives more info than we need, e.g


https://github.com/tbf-challenge/teamwork-be/blob/78e030a1b691c807b6a109ab09c9c1978f0cbaa7/src/routes/gifs/flag-gif.e2e.test.js#L62-L72

Reply:
@jocrah It works fine now. Thank you. I have made the changes